src/FlipCounter.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
```

Comment:
Let's start to learn how to connect and use tools, that we use inside our company.

And the most cool and the most important one is Typescript. 

Typescript is super-duper cool, it is really a game changer. It brings type safety concerns to your JS code, so it solves most of JS problems.

So let's convert this project to Typescript. [There is an article on CRA oficial docs](https://create-react-app.dev/docs/adding-typescript) on how to do it.